cmd/release-controller-api/http_helper.go

```diff
@@ -248,7 +248,7 @@ func tableLink(config *releasecontroller.ReleaseConfig, tag imagev1.TagReference
 		}
 		if strings.Contains(tag.Name, "nightly") && inconsistencies {
 			return fmt.Sprintf(`<td class="text-monospace"><a class="%s" href="/releasestream/%s/release/%s">%s</a> <a href="/releasestream/%s/inconsistency/%s"><i title="Inconsistency detected! Click for more details" class="bi bi-exclamation-circle"></i></a></td>`, phaseAlert(tag), template.HTMLEscapeString(config.Name), template.HTMLEscapeString(tag.Name), template.HTMLEscapeString(tag.Name), template.HTMLEscapeString(config.Name), template.HTMLEscapeString(tag.Name))
-		} else if config.Name == "4-stable" {
+		} else if config.As == "Stable" {
```

Comment:
Better to use: `releasecontroller.ReleaseConfigModeStable`

Reply:
Done